website_sale_product_detail_attribute_image/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module extends the functionality of website sale module to allow display
+product attributes images in shop product detail.
```

Comment:
s/product attributes images in shop product detail/images related to product attributes in e-commerce product page